packages/back-end/src/routers/demo-datasource-project/demo-datasource-project.controller.ts

```diff
@@ -375,6 +357,28 @@ export const postDemoDatasourceProject = async (
         });
       })
     );
+    // todo: save the experiment IDs
+
+    // Create feature
+    await Promise.all(
+      DEMO_FEATURES.map(async (f, i) => {
+        const featureToCreate: FeatureInterface = {
+          ...f,
+          id: getDemoDataSourceFeatureId(),
```

Comment:
If we're hardcoding the id, DEMO_FEATURES should not be an array.  Feature ids must be unique in an org, so adding more than one feature will break the code.

Same for DEMO_EXPERIMENTS.  We're hard-coding the trackingKey, which also should be unique per org.

Reply:
Yeah. We can just change them not to be arrays for now (they were arrays originally because there were more than 1, but we can re-add that complexity if and when we need it).

Reply:
So what's the requested change here? Remove the `DEMO_FEATURES` constant and inline the feature creation?

Reply:
Yeah, that's probably easiest.  Or change it to `DEMO_FEATURE` and it's just a single feature, not an array.  But inlining is simpler.  We can refactor later if we decide to add more features

Reply:
Same thing for `DEMO_EXPERIMENTS`, either inline or make it a single experiment, not an array

Reply:
This was addressed [here](https://github.com/growthbook/growthbook/pull/1499#issuecomment-1688972938).